server/consumer.go

```diff
@@ -1362,7 +1362,7 @@ func (o *consumer) deleteNotActive() {
 					js.mu.RUnlock()
 					if ca != nil {
 						if fs {
-							s.Warnf("Consumer assignment not cleaned up, retrying")
+							s.Warnf("Consumer assignment not cleaned up, for account: '%s', stream: '%s', consumer: '%s', retrying", acc, stream, name)
```

Comment:
Would remove ',' after "cleaned up", otherwise looks good.

Reply:
Ah I didn't reread it fully after changing it, done